buildSrc/src/main/kotlin/BuildConfigFields.kt

```diff
@@ -6,6 +6,12 @@ object BuildConfigFields {
         "\"APPCENTER_KEY\""
     )
 
+    val API_KEY = Field(
```

Comment:
Too generic name. Should be specific to it's purpose.

Reply:
Changed it to `IMAGE_API_KEY`.